swagger/swagger.json

```diff
@@ -1167,19 +1167,19 @@
         },
         "finished" : {
           "type" : "boolean",
-          "description" : "Whether the payment has finished",
+          "description" : "States whether the payment has finished",
           "readOnly" : true
         },
         "message" : {
           "type" : "string",
           "example" : "User cancelled the payment",
-          "description" : "What went wrong with the Payment if it finished with an error - English message",
+          "description" : "English message saying what went wrong with the Payment if it finished with an error",
```

Comment:
Can it be `Message saying what went wrong with the Payment if it finished with an error` ?

Reply:
Yes